mysql_tests/test_bind.py

```diff
@@ -0,0 +1,65 @@
+import random
+
+import pytest
+from gino.exceptions import UninitializedError
+from sqlalchemy.engine.url import make_url
+
+from .models import db, MYSQL_URL, User
+
+pytestmark = pytest.mark.asyncio
+
+
+# noinspection PyUnusedLocal
+async def test_create(bind):
+    nickname = "test_create_{}".format(random.random())
+    u = await User.create(nickname=nickname)
+    assert u.id is not None
+    assert u.nickname == nickname
+    return u
+
+
+async def test_get(bind):
+    u1 = await test_create(bind)
+    u2 = await User.get(u1.id)
+    assert u1.id == u2.id
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/python-gino/gino/pullRequest?prid=5587950)